nixos/modules/config/users-groups.nix

```
@@ -753,6 +753,14 @@ in
       '';
     };
 
+    users.defaultHome = mkOption {
```

I think that the name is a bit missleading, if I saw `defaultHome` I thought that would apply to all system users and not just normal users. Perhaps we can use the same naming that `defaultUserShell` does. So it might look more like:
```suggestion
    users.defaultUserHome = mkOption {
```